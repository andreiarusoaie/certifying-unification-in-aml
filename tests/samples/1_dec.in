variables: x, z
symbols: f, g , u, t
problem: f(x, g(z, x)) =? f(x, g(u, t))
