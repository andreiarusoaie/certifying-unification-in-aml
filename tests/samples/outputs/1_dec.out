Found Maude version: 3.0beta1

[1m[95mProof of:[94m (f(x, g(u, t)) or f(x, g(z, x)) <--> âˆƒ v(8) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), g(v(10), v(11))))  [0m
(1)(âˆƒ v(5) . (f(x, g(z, x)) <--> v(5) and (v(5) === f(x, g(z, x))))) [e-subst] ;

(2)((f(x, g(z, x)) <--> âˆƒ v(5) . v(5) and (v(5) === f(x, g(z, x)))) <--> âˆƒ v(5) . (f(x, g(z, x)) <--> v(5) and (v(5) === f(x, g(z, x))))) [e-scope] ;

(3)(f(x, g(z, x)) <--> âˆƒ v(5) . v(5) and (v(5) === f(x, g(z, x)))) [modus-ponens(1, 2)] ;

(4)(âˆƒ v(5) . (f(x, g(u, t)) <--> v(5) and (v(5) === f(x, g(u, t))))) [e-subst] ;

(5)((f(x, g(u, t)) <--> âˆƒ v(5) . v(5) and (v(5) === f(x, g(u, t)))) <--> âˆƒ v(5) . (f(x, g(u, t)) <--> v(5) and (v(5) === f(x, g(u, t))))) [e-scope] ;

(6)(f(x, g(u, t)) <--> âˆƒ v(5) . v(5) and (v(5) === f(x, g(u, t)))) [modus-ponens(4, 5)] ;

(7)(f(x, g(u, t)) or f(x, g(z, x)) <--> (âˆƒ v(5) . v(5) and (v(5) === f(x, g(u, t)))) or (âˆƒ v(5) . v(5) and (v(5) === f(x, g(z, x))))) [eqv-or(3, 6)] ;

(8)((âˆƒ v(5) . v(5) and (v(5) === f(x, g(u, t)))) or (âˆƒ v(5) . v(5) and (v(5) === f(x, g(z, x)))) <--> âˆƒ v(5) . v(5) and (v(5) === f(x, g(u, t))) or v(5) and (v(5) === f(x, g(z, x)))) [e-collapse] ;

(9)(v(5) and ((v(5) === f(x, g(u, t))) or (v(5) === f(x, g(z, x)))) <--> v(5) and (v(5) === f(x, g(u, t))) or v(5) and (v(5) === f(x, g(z, x)))) [and-or-distr] ;

(10)((âˆƒ v(5) . v(5) and ((v(5) === f(x, g(u, t))) or (v(5) === f(x, g(z, x))))) <--> âˆƒ v(5) . v(5) and (v(5) === f(x, g(u, t))) or v(5) and (v(5) === f(x, g(z, x)))) [e-context(9)] ;

(11)(f(x, g(u, t)) or f(x, g(z, x)) <--> âˆƒ v(5) . v(5) and (v(5) === f(x, g(u, t))) or v(5) and (v(5) === f(x, g(z, x)))) [eqv-tranz(7, 8)] ;

(12)(f(x, g(u, t)) or f(x, g(z, x)) <--> âˆƒ v(5) . v(5) and ((v(5) === f(x, g(u, t))) or (v(5) === f(x, g(z, x))))) [eqv-tranz(11, 10)] ;

(13)((v(5) === f(x, g(z, x))) <--> âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(z, x))) [e-gen] ;

(14)((v(5) === f(x, g(z, x))) <--> âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(z, x))) [and-eqv-intro(13)] ;

(15)((âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(z, x))) <--> âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(z, x))) [e-scope] ;

(16)((v(5) === f(x, g(z, x))) <--> âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(z, x))) [eqv-tranz(14, 15)] ;

(17)((v(5) === f(x, g(u, t))) <--> âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(u, t))) [e-gen] ;

(18)((v(5) === f(x, g(u, t))) <--> âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(u, t))) [and-eqv-intro(17)] ;

(19)((âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(u, t))) <--> âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(u, t))) [e-scope] ;

(20)((v(5) === f(x, g(u, t))) <--> âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(u, t))) [eqv-tranz(18, 19)] ;

(21)((v(5) === f(x, g(u, t))) or (v(5) === f(x, g(z, x))) <--> (âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(u, t))) or (âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(z, x)))) [eqv-or(16, 20)] ;

(22)((âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(u, t))) or (âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(z, x))) <--> âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(u, t)) or (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(z, x))) [e-collapse] ;

(23)((v(5) === f(x, g(u, t))) or (v(5) === f(x, g(z, x))) <--> âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(u, t)) or (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(z, x))) [eqv-tranz(21, 22)] ;

(24)(((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9))) <--> (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(u, t)) or (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(z, x))) [and-or-distr] ;

(25)((âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) <--> âˆƒ v(8) ; v(9) . (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(u, t)) or (v(5) === f(v(8), v(9))) and (v(8) === x) and (v(9) === g(z, x))) [e-context(24)] ;

(26)((v(5) === f(x, g(u, t))) or (v(5) === f(x, g(z, x))) <--> âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) [eqv-tranz(23, 25)] ;

(27)((âˆƒ v(5) . v(5) and ((v(5) === f(x, g(u, t))) or (v(5) === f(x, g(z, x))))) <--> âˆƒ v(5) . v(5) and (âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9))))) [e-context(26)] ;

(28)(v(5) and (âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) <--> âˆƒ v(8) ; v(9) . v(5) and ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) [e-scope] ;

(29)((âˆƒ v(5) . v(5) and (âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9))))) <--> âˆƒ v(5) . âˆƒ v(8) ; v(9) . v(5) and ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) [e-context(28)] ;

(30)((âˆƒ v(5) . âˆƒ v(8) ; v(9) . v(5) and ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) <--> âˆƒ v(5) ; v(8) ; v(9) . v(5) and ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) [e-set] ;

(31)((âˆƒ v(5) . v(5) and (âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9))))) <--> âˆƒ v(5) ; v(8) ; v(9) . v(5) and ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) [eqv-tranz(29, 30)] ;

(32)(((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (âˆƒ v(5) . v(5) and (v(5) === f(v(8), v(9)))) <--> âˆƒ v(5) . v(5) and ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) [e-scope] ;

(33)((âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (âˆƒ v(5) . v(5) and (v(5) === f(v(8), v(9))))) <--> âˆƒ v(8) ; v(9) . âˆƒ v(5) . v(5) and ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) [e-context(32)] ;

(34)((âˆƒ v(8) ; v(9) . âˆƒ v(5) . v(5) and ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) <--> âˆƒ v(5) ; v(8) ; v(9) . v(5) and ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) [e-set] ;

(35)((âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (âˆƒ v(5) . v(5) and (v(5) === f(v(8), v(9))))) <--> âˆƒ v(5) ; v(8) ; v(9) . v(5) and ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) [eqv-tranz(34, 33)] ;

(36)(âˆƒ v(5) . (f(v(8), v(9)) <--> v(5) and (v(5) === f(v(8), v(9))))) [e-subst] ;

(37)((f(v(8), v(9)) <--> âˆƒ v(5) . v(5) and (v(5) === f(v(8), v(9)))) <--> âˆƒ v(5) . (f(v(8), v(9)) <--> v(5) and (v(5) === f(v(8), v(9))))) [e-scope] ;

(38)(f(v(8), v(9)) <--> âˆƒ v(5) . v(5) and (v(5) === f(v(8), v(9)))) [modus-ponens(36, 37)] ;

(39)((âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and f(v(8), v(9))) <--> âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (âˆƒ v(5) . v(5) and (v(5) === f(v(8), v(9))))) [e-context(38)] ;

(40)((âˆƒ .Vars . âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and f(v(8), v(9))) <--> âˆƒ .Vars . âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (âˆƒ v(5) . v(5) and (v(5) === f(v(8), v(9))))) [e-context(39)] ;

(41)((âˆƒ .Vars . âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (âˆƒ v(5) . v(5) and (v(5) === f(v(8), v(9))))) <--> âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (âˆƒ v(5) . v(5) and (v(5) === f(v(8), v(9))))) [e-set] ;

(42)((âˆƒ .Vars . âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and f(v(8), v(9))) <--> âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and f(v(8), v(9))) [e-set] ;

(43)((âˆƒ .Vars . âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and f(v(8), v(9))) <--> âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (âˆƒ v(5) . v(5) and (v(5) === f(v(8), v(9))))) [eqv-tranz(41, 40)] ;

(44)((âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and f(v(8), v(9))) <--> âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (âˆƒ v(5) . v(5) and (v(5) === f(v(8), v(9))))) [eqv-tranz(43, 42)] ;

(45)((âˆƒ v(5) . v(5) and ((v(5) === f(x, g(u, t))) or (v(5) === f(x, g(z, x))))) <--> âˆƒ v(5) ; v(8) ; v(9) . v(5) and ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (v(5) === f(v(8), v(9)))) [eqv-tranz(27, 31)] ;

(46)((âˆƒ v(5) . v(5) and ((v(5) === f(x, g(u, t))) or (v(5) === f(x, g(z, x))))) <--> âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and (âˆƒ v(5) . v(5) and (v(5) === f(v(8), v(9))))) [eqv-tranz(45, 35)] ;

(47)((âˆƒ v(5) . v(5) and ((v(5) === f(x, g(u, t))) or (v(5) === f(x, g(z, x))))) <--> âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and f(v(8), v(9))) [eqv-tranz(44, 46)] ;

(48)(f(x, g(u, t)) or f(x, g(z, x)) <--> âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and f(v(8), v(9))) [eqv-tranz(12, 47)] ;

(49)((v(9) === g(z, x)) <--> âˆƒ v(10) ; v(11) . (v(9) === g(v(10), v(11))) and (v(10) === z) and (v(11) === x)) [e-gen] ;

(50)((v(8) === x) and (v(9) === g(z, x)) <--> (v(8) === x) and (âˆƒ v(10) ; v(11) . (v(9) === g(v(10), v(11))) and (v(10) === z) and (v(11) === x))) [and-eqv-intro(49)] ;

(51)((v(8) === x) and (âˆƒ v(10) ; v(11) . (v(9) === g(v(10), v(11))) and (v(10) === z) and (v(11) === x)) <--> âˆƒ v(10) ; v(11) . (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === z) and (v(11) === x)) [e-scope] ;

(52)((v(8) === x) and (v(9) === g(z, x)) <--> âˆƒ v(10) ; v(11) . (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === z) and (v(11) === x)) [eqv-tranz(50, 51)] ;

(53)((v(9) === g(u, t)) <--> âˆƒ v(10) ; v(11) . (v(9) === g(v(10), v(11))) and (v(10) === u) and (v(11) === t)) [e-gen] ;

(54)((v(8) === x) and (v(9) === g(u, t)) <--> (v(8) === x) and (âˆƒ v(10) ; v(11) . (v(9) === g(v(10), v(11))) and (v(10) === u) and (v(11) === t))) [and-eqv-intro(53)] ;

(55)((v(8) === x) and (âˆƒ v(10) ; v(11) . (v(9) === g(v(10), v(11))) and (v(10) === u) and (v(11) === t)) <--> âˆƒ v(10) ; v(11) . (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === u) and (v(11) === t)) [e-scope] ;

(56)((v(8) === x) and (v(9) === g(u, t)) <--> âˆƒ v(10) ; v(11) . (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === u) and (v(11) === t)) [eqv-tranz(54, 55)] ;

(57)((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x)) <--> (âˆƒ v(10) ; v(11) . (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === u) and (v(11) === t)) or (âˆƒ v(10) ; v(11) . (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === z) and (v(11) === x))) [eqv-or(52, 56)] ;

(58)((âˆƒ v(10) ; v(11) . (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === u) and (v(11) === t)) or (âˆƒ v(10) ; v(11) . (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === z) and (v(11) === x)) <--> âˆƒ v(10) ; v(11) . (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === z) and (v(11) === x)) [e-collapse] ;

(59)((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x)) <--> âˆƒ v(10) ; v(11) . (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === z) and (v(11) === x)) [eqv-tranz(57, 58)] ;

(60)(((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (v(9) === g(v(10), v(11))) <--> (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === z) and (v(11) === x)) [and-or-distr] ;

(61)((âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (v(9) === g(v(10), v(11)))) <--> âˆƒ v(10) ; v(11) . (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(9) === g(v(10), v(11))) and (v(10) === z) and (v(11) === x)) [e-context(60)] ;

(62)((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x)) <--> âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (v(9) === g(v(10), v(11)))) [eqv-tranz(59, 61)] ;

(63)((âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and f(v(8), v(9))) <--> âˆƒ v(8) ; v(9) . f(v(8), v(9)) and (âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (v(9) === g(v(10), v(11))))) [e-context(62)] ;

(64)(f(v(8), v(9)) and (âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (v(9) === g(v(10), v(11)))) <--> âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) [e-scope] ;

(65)((âˆƒ v(8) ; v(9) . f(v(8), v(9)) and (âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (v(9) === g(v(10), v(11))))) <--> âˆƒ v(8) ; v(9) . âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) [e-context(64)] ;

(66)((âˆƒ v(8) ; v(9) . âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) <--> âˆƒ v(8) ; v(9) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) [e-set] ;

(67)((âˆƒ v(8) ; v(9) . f(v(8), v(9)) and (âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (v(9) === g(v(10), v(11))))) <--> âˆƒ v(8) ; v(9) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) [eqv-tranz(65, 66)] ;

(68)(((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (âˆƒ v(9) . f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) <--> âˆƒ v(9) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) [e-scope] ;

(69)((âˆƒ v(8) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (âˆƒ v(9) . f(v(8), v(9)) and (v(9) === g(v(10), v(11))))) <--> âˆƒ v(8) ; v(10) ; v(11) . âˆƒ v(9) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) [e-context(68)] ;

(70)((âˆƒ v(8) ; v(10) ; v(11) . âˆƒ v(9) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) <--> âˆƒ v(8) ; v(9) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) [e-set] ;

(71)((âˆƒ v(8) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (âˆƒ v(9) . f(v(8), v(9)) and (v(9) === g(v(10), v(11))))) <--> âˆƒ v(8) ; v(9) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) [eqv-tranz(70, 69)] ;

(72)(âˆƒ v(9) . (f(v(8), g(v(10), v(11))) <--> f(v(8), v(9)) and (v(9) === g(v(10), v(11))))) [e-subst] ;

(73)((f(v(8), g(v(10), v(11))) <--> âˆƒ v(9) . f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) <--> âˆƒ v(9) . (f(v(8), g(v(10), v(11))) <--> f(v(8), v(9)) and (v(9) === g(v(10), v(11))))) [e-scope] ;

(74)(f(v(8), g(v(10), v(11))) <--> âˆƒ v(9) . f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) [modus-ponens(72, 73)] ;

(75)((âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), g(v(10), v(11)))) <--> âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (âˆƒ v(9) . f(v(8), v(9)) and (v(9) === g(v(10), v(11))))) [e-context(74)] ;

(76)((âˆƒ v(8) . âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), g(v(10), v(11)))) <--> âˆƒ v(8) . âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (âˆƒ v(9) . f(v(8), v(9)) and (v(9) === g(v(10), v(11))))) [e-context(75)] ;

(77)((âˆƒ v(8) . âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (âˆƒ v(9) . f(v(8), v(9)) and (v(9) === g(v(10), v(11))))) <--> âˆƒ v(8) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (âˆƒ v(9) . f(v(8), v(9)) and (v(9) === g(v(10), v(11))))) [e-set] ;

(78)((âˆƒ v(8) . âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), g(v(10), v(11)))) <--> âˆƒ v(8) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), g(v(10), v(11)))) [e-set] ;

(79)((âˆƒ v(8) . âˆƒ v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), g(v(10), v(11)))) <--> âˆƒ v(8) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (âˆƒ v(9) . f(v(8), v(9)) and (v(9) === g(v(10), v(11))))) [eqv-tranz(77, 76)] ;

(80)((âˆƒ v(8) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), g(v(10), v(11)))) <--> âˆƒ v(8) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (âˆƒ v(9) . f(v(8), v(9)) and (v(9) === g(v(10), v(11))))) [eqv-tranz(79, 78)] ;

(81)((âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and f(v(8), v(9))) <--> âˆƒ v(8) ; v(9) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), v(9)) and (v(9) === g(v(10), v(11)))) [eqv-tranz(63, 67)] ;

(82)((âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and f(v(8), v(9))) <--> âˆƒ v(8) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and (âˆƒ v(9) . f(v(8), v(9)) and (v(9) === g(v(10), v(11))))) [eqv-tranz(81, 71)] ;

(83)((âˆƒ v(8) ; v(9) . ((v(8) === x) and (v(9) === g(u, t)) or (v(8) === x) and (v(9) === g(z, x))) and f(v(8), v(9))) <--> âˆƒ v(8) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), g(v(10), v(11)))) [eqv-tranz(80, 82)] ;

(84)(f(x, g(u, t)) or f(x, g(z, x)) <--> âˆƒ v(8) ; v(10) ; v(11) . ((v(8) === x) and (v(10) === u) and (v(11) === t) or (v(8) === x) and (v(10) === z) and (v(11) === x)) and f(v(8), g(v(10), v(11)))) [eqv-tranz(48, 83)] ;
[94mChecked: [0m [1m[92m true
[0m
